Fix missing finding aid logo, refs #13458

djjuhasz opened the pull request:
- Fix invalid FO XSL template elements
- Add "{{ app_root }}" template variables to XSL files
- Add "{{ app_root }}" to helper-functions.xsl href
- Replace template vars with AtoM application path at run time

mcantelon reviewed (approved):
One line in one of the commit messages has a typo (maybe "in" rather that "to">): `"Use {{ app_root }} to helper-functions.xsl href`.

Otherwise looks good to me! The `app_root` tag it a nice touch.

djjuhasz committed:
Fix missing finding aid logo, refs #13458

- Add "{{ app_root }}" template variables to pdf-logo href
- Add "{{ app_root }}" to helper-functions.xsl href
- Replace template vars with AtoM application path at run time
- Cosmetic XSL format changes to indenting, etc.

djjuhasz committed:
Cosmetic: reformat XSL docs with xmllint

xmllint command:

 export XMLLINT_INDENT="    "
 find lib/task/pdf/ -name "*.xsl" -type f -exec xmllint --output '{}' \
 --format --noblanks '{}' \;

After which I manually found and replaced extra whitespaces (more than
one space) between words [i.e. "(\S)  +(\S)" -> "$1 $2"].
